network-api/networkapi/wagtailpages/templates/fragments/product_tab.html

```diff
@@ -87,7 +87,11 @@ <h3 class="tw-mb-4 tw-font-zilla tw-text-2xl tw-leading-7">{% trans "What data d
 
                     {% trans "What is the company’s known track record of protecting users’ data?" as company_track_record_label %}
                     {% include "fragments/product_criterion.html"  with  label=company_track_record_label  value=product.company_track_record|track_record   help=product.track_record_details  ding=product.track_record_is_bad rich_help_text=True %}
-
+                    
+                    {% if product.child_safety_blurb %}
+                    {% trans "Child Privacy Information" as child_privacy_label %}
+                    {% include "fragments/product_criterion.html"  with  label=child_privacy_label  value=product.child_safety_blurb rich_text_value=True  %}
```

Comment:
**Suggestion** I find templates usually easier to read when we apply indention rules to them similar to Python. E.g. these two lines should be indented, because they are inside an if-block.